whylabs_toolkit/monitor/manager/monitor_setup.py

```diff
@@ -151,6 +157,23 @@ def mode(self) -> Optional[Union[EveryAnomalyMode, DigestMode]]:
     def mode(self, mode: Union[EveryAnomalyMode, DigestMode]) -> None:
         self._monitor_mode = mode
 
+    @property
+    def is_constraint(self) -> Optional[bool]:
+        return not (self._analyzer_tags is None) and TAG_ANALYZER_CONSTRAINT in self._analyzer_tags
```

Comment:
a bit confusing to have this laid out as is. 
would rewrite for better understanding as:
`self._analyzer_tags is not None and TAG_ANALYZER_CONSTRAINT in self._analyzer_tags`